test/cases/testdata/globmatch/test-globmatch-0159.yaml

```diff
@@ -0,0 +1,15 @@
+cases:
+- data: {}
+  modules:
+  - |
+    package generated
+
+    p[x] {
+      glob.match("*", null, "foo.bar", x)
+    }
+  note: globmatch/glob match single without default delimiter
+  query: data.generated.p = x
+  sort_bindings: true
+  want_result:
+  - x:
+    - true
```

Comment:
Would you mind adding one case like
```rego
glob.match("foo*", null, "foo.bar", x)
```

just to make the "no delimiters" case more obvious?

Reply:
I have added another test case to more clearly about this.